go/integTest/batch_test.go

```diff
@@ -127,7 +127,17 @@ func (suite *GlideTestSuite) TestBatchMove() {
 		key := "{prefix}-" + uuid.NewString()
 		switch c := client.(type) {
 		case *glide.ClusterClient:
-			return // Move is not supported in cluster client
+			if suite.serverVersion >= "9.0.0" {
+				batch := pipeline.NewClusterBatch(isAtomic).
+					Set(key, "val").
+					Move(key, 2)
+
+				res, err := c.Exec(context.Background(), *batch, true)
+				suite.verifyOK(res[0].(string), err)
+				suite.True(res[1].(bool))
+			} else {
+				return // Move is not supported in cluster client
```

Comment:
This doesn't make sense. Is move supported in Clusters or not? It seems like this comment should indicate that it is not supported in cluster client before Valkey 9.

Reply:
refactored to skip the test if the version is lower than 9 for for cluster mode